fedora/python-ogr.spec

```diff
@@ -1,7 +1,7 @@
 %global srcname ogr
 
 Name:           python-%{srcname}
-Version:        0.40.0
+Version:        0.40.1
```

Comment:
Why 0.40.1 instead of 0.41.0? This doesn't seem like a patch-sized bugfix

Reply:
Ok, I thought it was a minor fix